CHANGES.rst

```diff
@@ -5,7 +5,8 @@
 3.0.1 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Adapt tests to `zope.configuration >= 4.2`.
```

Comment:
Because of the install requires minor bump of the dependency, CHANGES.rst and setup.py should go to 3.1.0 here.

Reply:
Done